docs/thirdparty-openapi3-snippets.yaml

```diff
@@ -1748,7 +1748,7 @@ components:
       type: object
       description: The object sent in a `POST /consents` request.
       properties:
-        id:
+        consentId:
           allOf:
             - title: CorrelationId
```

Comment:
why not refer v1_1.CorrelationId ?

Reply:
This doc is generated for browsing purposes. It does bundle from v1_1.